src/bench/block_assemble.cpp

```diff
@@ -39,7 +39,7 @@ static void AssembleBlock(benchmark::State& state)
         LOCK(::cs_main); // Required for ::AcceptToMemoryPool.
 
         for (const auto& txr : txs) {
-            CValidationState state;
+            TxValidationState state;
```

Comment:
In commit "[validation] Add CValidationState subclasses" (1faf0c4b0120995b4361d38ca0cc0114e633955d)

Note to other reviewers: This first commit is huge and can be easier to start reviewing if you revert the rote, mechanical changes:

```sh
git checkout 1faf0c4b0120995b4361d38ca0cc0114e633955d
git grep -l ValidationState | xargs sed -i 's/BlockValidationState\|TxValidationState/CValidationState/g'
git grep -l ValidationResult | xargs sed -i 's/BlockValidationResult\|TxValidationResult/ValidationInvalidReason/g'
git grep -l MaybePunish | xargs sed -i 's/MaybePunishNode\(ForBlock\|ForTx\)/MaybePunishNode/g'
git diff HEAD^
```

After that it's possible to review the full change with:

```sh
git log -p -n1 -U0 --word-diff-regex=. 1faf0c4b0120995b4361d38ca0cc0114e633955d
```

Reply:
Thanks Russ. I've added that review tip to the PR description.